### ISSUE TITLE
Feature for child bridge

### ISSUE THREAD

**Issue by dadsalleb** [tmyrden/homebridge-dummy-sensor#6]
The original `homebridge-dummy` plugin has a feature for the child bridge but individual switches are acting like a child bridge. Instead, I would like to have a child bridge feature to restart the `homebridge-dummy-sensor` only.

**Comment by tmyrden** [comment]
It is currently possible to turn each `homebridge-dummy-sensor` into it's own child bridge, but not the whole plugin itself.

Just need to clarify:

Are you having trouble with child bridging each switch/sensor combo or are you wanting that ability to child bridge the entire plugin?

**Comment by tmyrden** [comment]
Correct. This is the difference between registering your plugin as a `platform` vs a number of `accessory` types. I will look into what it would take to switch over to a platform eventually - this is already proving to be a pain point during development since I have to restart the whole bridge with every change I make to the plugin.
